pytorch_toolkit/formula_recognition/tools/demo.py

```diff
@@ -60,9 +59,15 @@ def parse_args():
     with open(args.config, 'r') as f:
         config = yaml.load(f, Loader=yaml.SafeLoader).get("demo")
     demo = Im2latexDemo(config)
-    for inp in config.get('input_images'):
-        img_path = os.path.join(demo.root_dir, inp)
-        input_image = cv.imread(img_path, cv.IMREAD_COLOR)
-        assert input_image is not None, "Error reading image {}, please, check input path".format(img_path)
+    if os.path.isdir(args.input):
+        inputs = sorted(os.path.join(args.input, inp)
+                        for inp in os.listdir(args.input))
+    else:
+        inputs = [args.input]
+    for inp in inputs:
+        if not os.path.isabs(inp):
+            inp = os.path.join(demo.root_dir, inp)
```

Comment:
This is counterintuitive. When somebody runs the script, she/he provides the path which is relative to the current working directory, not the script directory.

Reply:
Fixed